clippy_lints/src/attrs.rs

```diff
@@ -417,11 +417,14 @@ fn check_attrs(cx: &LateContext<'_, '_>, span: Span, name: Name, attrs: &[Attrib
     }
 
     for attr in attrs {
-        if attr.is_sugared_doc {
-            return;
-        }
+        let attr_item = if let AttrKind::Normal(ref attr) = attr.kind {
+            attr
+        } else {
+            continue;
```

Comment:
the old diff is `return`, this one is `continue`. Which one is correct?

Reply:
(Through I think this one is more meaningful.)

Reply:
I think `continue` is better here